lightkurve/utils.py

```diff
@@ -293,3 +299,67 @@ def plot_image(image, ax=None, scale='linear', origin='lower',
     if show_colorbar:
         plt.colorbar(cax, ax=ax, norm=norm, label=clabel)
     return ax
+
+
+
+def query_catalog(coordinate, catalog="KIC", radius=0.5):
+        """
+        Returns an astropy table of the sources inside a radius query.
+        Current catalogs supported are KIC, EPIC and Gaia DR2.
+
+        Parameters
+        -----------
+        coordinate : astropy.coordinates.SkyCoord
+            Coordinate to query around.
+        catalog: 'KIC', 'EPIC', or 'Gaia'
+            Indicate catalog assigned for mission.
+        radius: float
+            Radius of cone search centered on the target in arcminutes.
+            Default radius is 0.5 arcmin.
+
+        Returns
+        -------
+        result : astropy.table
+            Astropy table with the following columns
+            ID : Catalog ID from catalog.
+            RAJ200: Right ascension [degrees]
+            DEJ2000: Declination [degrees]
+            pmRA: Proper motion for right ascension [mas/year]
+            pmDEC: Proper motion for declination [mas/year]
+            Kpmag: Magnitude in Kepler band [mag]
+        """
+        supported_catalogs = ['KIC', 'EPIC', 'Gaia']
```

Comment:
Very minor nitpick, rather than having two variables that store the names of the catalog we could have just one (`ID`) which you could print using...

`ValueError('catalog not one of {}'.format([key for key in ID.keys()]))`

You'd just need to move `ID` above this statement.